containers/ddev-webserver/tests/ddev-webserver/general.bats

```diff
@@ -31,6 +31,11 @@
     [[ "${v}" > "2." ]]
 }
 
+@test "verify that PHP assertion are enabled by default" {
+    enabled=$(docker exec $CONTAINER_NAME sh -c "php -r 'echo ini_get("'"zend.assertions"'");'")
+    [[ "${enabled}" -eq 1 ]]
+}
+
```

Comment:
Looks like this was successful, https://github.com/drud/ddev/pull/2833/checks?check_run_id=1990643272#step:5:1378

Congrats!